lib/ansible/modules/rpm_key.py

```diff
@@ -203,6 +204,33 @@ def getfingerprint(self, keyfile):
 
         self.module.fail_json(msg="Unexpected gpg output")
 
+    def getfingerprints(self, keyfile):
```

Comment:
I was not sure whether `getfingerprint` is exported or used elsewhere.

To avoid conflict, I added this method and swapped out the invocation in this module.

If this is internal only, then I can remove the `getfingerprint` method.

Reply:
The `getfingerprint` method is internal to this module.

Reply:
Excellent. I made some changes:

- removed the `getfingerprint` method in favor of the pluralized version
- added ability to verify either the primary or subkey fingerprint or require both
- added changelog fragment

I think this makes it ready for an initial review. I I opted not to squash in the changes to allow reviewers to see my evolution of thought.

If this request is accepted, they should be squashed.